Docs.md

```diff
@@ -337,7 +337,22 @@ https://github.com/denoland/deno/blob/master/src/msg.fbs
 See
 [CONTRIBUTING.md](https://github.com/denoland/deno/blob/master/.github/CONTRIBUTING.md).
 
-## Change Log
+## Changelog
+
+### 2018.11.27 / v0.2.0 / Mildly usable
+
+The resources table was added to abstract various types of I/O streams and other
```

Comment:
This is not a user visible change - maybe place it at the end?

Reply:
fixed